src/mscorlib/model.xml

```diff
@@ -3413,6 +3413,7 @@
       <Member Name="GetHashCode" />
       <Member Name="GetInvocationList" />
       <Member Name="GetMethodImpl" />
+      <Member Name="GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
```

Comment:
I'm just curious.  It's fine to be explicit about it, but this was already getting output in the DLL due to it being the implementation of an interface method, right?